travis/travis_after_tests_success

```diff
@@ -19,3 +22,12 @@ if (os.environ.get('TESTS', '1') == '1' and
         codecov_main(argv=None)
     except:
         pass
+
+if os.environ.get('DOC') == '1':
```

Comment:
Not a blocker, but can we name this `AUTODOC` instead? It'll jive a little better with pre-existing jargon IMO.